packages/client/components/AddTeamDialog.tsx

```diff
@@ -84,6 +85,12 @@ const AddTeamDialog = (props: Props) => {
   }
 
   const onSelectedUsersChange = (newUsers: Option[]) => {
+    if (newUsers.length && newUsers.length > selectedUsers.length) {
+      SendClientSegmentEventMutation(atmosphere, 'Teammate Selected', {
+        selectionLocation: 'addTeamUserPicker'
+      })
+    }
```

Comment:
+1 I don't really think it matters, but we typically send events at the end of the relevant function after the work is done.